console/src/components/Services/Settings/Actions.ts

```diff
@@ -185,8 +199,11 @@ export const replaceMetadataFromFile = (
   dispatch(replaceMetadata(parsedFileContent, onSuccess, onError));
 };
 
-const handleInconsistentObjects = inconsistentObjects => {
-  return (dispatch, getState) => {
+const handleInconsistentObjects = (inconsistentObjects: MetadataObject[]) => {
+  return (
+    dispatch: ThunkDispatch<any, any, AnyAction>,
+    getState: () => AppState
```

Comment:
At this point, you would have to _merge_ `AppState` to `ReduxState` type to add what's missing in `ReduxState`.

```suggestion
    getState: GetReduxState
```